pages/docs/form/form-control.mdx

```diff
@@ -22,6 +22,17 @@ forms.
 
 ## Import
 
+Chakra UI export 4 components for Form Control:
+
+- **`FormControl`**: The wrapper that provides context and functionality for all
+  children.
+- **`FormLabel`**: The label of a form section. The usage is similar to
+  [html label](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label).
+- **`FormHelperText`**: The message that tells users more detail things about
+  the form section.
+- **`FormErrorMessage`**: The message that shows up when an error occurs and
+  tells users how to make it right.
```

Comment:
Let's write just "The message that shows up when an error occurs", without the "and tells users how to make it right" part, because the error message is just a plain text defined from the user itself.